static/js/query-configs/websites.ts

```diff
@@ -235,57 +236,71 @@ export const createWebsiteCollaboratorMutation = (
   }
 }
 
-export type WebsiteContentListingResponse = CollectionResponse<
+export type WebsiteContentListingResponse = PaginatedResponse<
   WebsiteContentListItem
 >
 type WebsiteContentListing = Record<string, string[]> // website name to list of uuids
 export const contentListingKey = (
-  name: string,
-  type: string,
-  offset: number
-): string => JSON.stringify([name, type, offset])
+  listingParams: ContentListingParams
+): string => JSON.stringify(Object.values(listingParams))
```

Comment:
I think we need to specify the values to be stringified explicitly so that the key doesn't change with a different insertion order for `listingParams`. Otherwise you could have something like `[ name, type, offset ]` and `[type, name, offset]` which are two different keys for the same object.